packages/util/package.json

```diff
@@ -28,6 +28,15 @@
   },
   "homepage": "https://github.com/avantcredit/gql2ts#readme",
   "dependencies": {
+    "glob": "^7.1.2",
+    "graphql": "^0.10.1",
+    "graphql-tag": "^2.2.2",
+    "graphql-tools": "^1.0.0",
+    "merge-graphql-schemas": "0.0.17"
```

Comment:
some of those were merge-graphql-schemas peer dependencies, will see how to get rid of them.